src/pages/Todo/Todo.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useState } from "react";
+import { Container, Typography } from "@mui/material";
+import { v4 as uuidv4 } from "uuid";
+import Header from "./components/Header";
+import TodoList from "./components/TodoList";
+import { Task } from "./types";
+import { useSnackbarAlerts } from "../../hooks/useSnackbarAlerts";
+
+const Todo: FC = () => {
+  const [todoList, setToDoList] = useState<Task[]>([]);
+  const { showSuccessSnackbar } = useSnackbarAlerts();
+
+  const addTask = (taskName: string, deadline: number): void => {
+    const newTask: Task = {
+      id: uuidv4(),
+      taskName,
+      deadline,
+      isComplete: false,
+    };
+    setToDoList([...todoList, newTask]);
```

Comment:
Use previous value when using useState it will take the last updated state and will not give yo unexpected results. 
```suggestion
    setToDoList(prev=>[...prev, newTask]);
```